src/fetch/payouts.py

```diff
@@ -401,10 +399,19 @@ def construct_payout_dataframe(
     normalize_address_field(service_fee_df, join_column)
 
     # 3. Merge the three dataframes (joining on solver)
+
+    reward_target_reduced_df_columns = [
+        x for x in list(reward_target_df.columns) if x != "solver_name"
+    ]
+    reward_target_reduced_df = reward_target_df[reward_target_reduced_df_columns]
+    service_fee_reduced_df_columns = [
+        x for x in list(service_fee_df.columns) if x != "solver_name"
+    ]
+    service_fee_reduced_df = service_fee_df[service_fee_reduced_df_columns]
```

Comment:
I suppose that we are not really interested in the fully general approach. Then this would work:
```suggestion
    reward_target_reduced_df = reward_target_df.drop("solver_name", axis=1)
    service_fee_reduced_df = service_fee_df.drop("solver_name", axis=1)
```
If we actually want that generality, this and other data transformations should probably happen somewhere else. E.g. right after fetching the data.